app/controllers/api/v1/messages_controller.rb

```diff
@@ -1,6 +1,14 @@
 module Api
   module V1
     class MessagesController < ApiController
+      def index
+        # TODO: Add Pagination and order filter
+        return render status: 404, nothing: true unless chat_room
+
+        messages = Message.where(chat_room: chat_room).order(created_at: :asc)
```

Comment:
I think that correct order here is descending, but it doesn't matter so much unless we add pagination.